chain/chunks/src/lib.rs

```diff
@@ -481,6 +481,7 @@ pub struct ShardsManager {
     encoded_chunks: EncodedChunksCache,
     requested_partial_encoded_chunks: RequestPool,
     chunk_forwards_cache: lru::LruCache<ChunkHash, HashMap<u64, PartialEncodedChunkPart>>,
+    chain_head: Option<Tip>,
```

Comment:
Add comment that this is a "best effort" and updated asynchronously, and point to where the source of truth is?

I think it's useful to explicitly mark "this is not source of truth" bit

Reply:
Agreed! Added a comment.